package.json

```diff
@@ -11,10 +11,15 @@
     "postinstall": "lerna run build --scope=vibe-storybook-components"
   },
   "devDependencies": {
+    "@types/react-color": "^3.0.12",
     "lerna": "^8.1.2",
     "typescript": "^4.7.3"
   },
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "dependencies": {
+    "react-color": "^2.19.3",
+    "react-colorful": "^5.6.1"
+  }
```

Comment:
These should be in the `core` package's, this is the root package of the monorepo so deps here apply to all packages